lib/Test/Nginx/Util.pm

```diff
@@ -78,6 +81,19 @@ our $ServerAddr = '127.0.0.1';
 
 our $ServerName = 'localhost';
 
+our $Http3SSLCrt = $ENV{TEST_NGINX_HTTP3_CRT};
+our $Http3SSLCrtKey = $ENV{TEST_NGINX_HTTP3_CRT_KEY};
+
+if ($UseHttp2 && $UseHttp3) {
+    die "Ambiguous: both TEST_NGINX_USE_HTTP3 and TEST_NGINX_USE_HTTP2 are set.\n";
+}
+
+if ($UseHttp3 && (!defined $Http3SSLCrt || !defined $Http3SSLCrtKey)) {
```

Comment:
We shouldn't force the user to set the path to the certificate, they can load the certificate in other ways: perhaps by modifying the `http_config` or `config` block, perhaps by loading the certificate by lua

Reply:
This feature allows the user to run all tests in HTTP/3 without modifying individual .t test case files.